scripts/pre_commit_linter.py

```diff
@@ -347,6 +347,15 @@
     '/scripts/install_third_party.sh',
     '/.github/']
 
+OLD_CONVENTION_DIRECTORIES = [
+    'core/templates/dev/head/services',
+    'core/templates/dev/head/base_components',
```

Comment:
Isn't this new? It wasn't around in the "old" codebase...

Reply:
I don't think I understood you completely. These are the directories which follow the old convention and hence the name.

Reply:
I guess I was thinking that base_components seems to have been recently added and is here to stay? I thought the idea was that all "new" dirs are cleaned up and we were going to remove all the dirs in this list completely over time... (which, btw, merits an issue or a TODO).

Are we planning to delete base_components too?

Reply:
No, it doesn't need to be deleted. It just means that it is following the old convention of file naming. When we will empty this list out, it would mean that these directories now follow the new convention.

Reply:
OK. I think you missed adding both an issue and a TODO for this (see previous comment). We need some way of keeping track of the task of "making this list empty".

Also add a comment above line 350 to explain to new devs what this constant is (in particular what is meant by "old convention").

Reply:
I have added a comment as well a TODO. PTAL.